apps/web/src/pages/Applications/ApplicationSkillsPage/ApplicationSkillsPage.stories.tsx

```diff
@@ -1,37 +1,38 @@
 import React from "react";
-import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Meta, StoryFn } from "@storybook/react";
 
 import {
-  fakeExperiences,
   fakePoolCandidates,
+  fakeExperienceForDate,
 } from "@gc-digital-talent/fake-data";
 
 import { notEmpty } from "@gc-digital-talent/helpers";
-import { ApplicationSkills } from "./ApplicationSkillsPage";
-import { ApplicationPageProps } from "../ApplicationApi";
+import {
+  ApplicationSkills,
+  ApplicationSkillsProps,
+} from "./ApplicationSkillsPage";
 
 const fakePoolCandidate = fakePoolCandidates(1)[0];
 const fakeUser = fakePoolCandidate.user;
-const mockExperiences = fakeExperiences(5);
-const mockPoolSkills =
-  fakePoolCandidate?.pool.essentialSkills?.filter(notEmpty);
+const mockExperiences = fakeExperienceForDate(5);
 const experienceSkills = mockExperiences
   .filter(notEmpty)
   .map((experience) => experience.skills)
   .filter(notEmpty)
   .flatMap((skill) => skill);
 
-const noSkills: ApplicationPageProps = {
+const noSkills: ApplicationSkillsProps = {
   application: {
     ...fakePoolCandidate,
     user: {
       ...fakeUser,
       experiences: mockExperiences,
     },
   },
+  experiences: mockExperiences,
 };
```

Comment:
I see you added it, it is just the `startDate` and `endDate` that is causing problems I assume? I'm wondering why we tacked that on to the `AwardExperience` since they do not exist there 🤔 

I think this is more of a bug than simple "stories are not working" 😬

Reply:
Hm, what do you think should be done?

Reply:
We have the `AnyExperience` type that is essentially the same as `ExperienceForDate` but does not add `startDate` and `endDate` to `AwardExperience`. We could just use that type instead and then it should be as easy as adding `experiences` to the top level as well.

Reply:
Ah, okay, I made some changes. Is it now what you are thinking of?